clerk/instances.go

```diff
@@ -64,13 +64,15 @@ type InstanceRestrictionsResponse struct {
 	Blocklist                   bool   `json:"blocklist"`
 	BlockEmailSubaddresses      bool   `json:"block_email_subaddresses"`
 	BlockDisposableEmailDomains bool   `json:"block_disposable_email_domains"`
+	IgnoreDotsForGmailAddresses bool   `json:"ignore_dots_for_gmail_addresses,omitempty"`
```

Comment:
🔧 You probably don't need `omitempty` in this context, since this is not serialized. 

```suggestion
	IgnoreDotsForGmailAddresses bool   `json:"ignore_dots_for_gmail_addresses"`
```